test/e2e/coverage_docs/resource_test.md

```diff
@@ -0,0 +1,13 @@
+### resource e2e test coverage analysis
+
+| Test Case                                                                                                                       | E2E Describe Text               | Comments                                                                                                             |
+|---------------------------------------------------------------------------------------------------------------------------------|---------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| DeploymentStatus collection testing: Test the status collection of Deployment resources.                                        | DeploymentStatus collection     |                                                                                                                      |
+| ServiceStatus collection testing: Test the status collection of Service resources.                                              | ServiceStatus collection        |                                                                                                                      |
+| NodePort Service collection testing: Test the status collection of NodePort type Service resources.                             | NodePort Service collection     |                                                                                                                      |
+| IngressStatus collection testing: Test the status collection of Ingress resources.                                              | IngressStatus collection        |                                                                                                                      |
+| JobStatus collection testing: Test the status collection of Job resources.                                                      | JobStatus collection            |                                                                                                                      |
+| DaemonSetStatus collection testing: Test the status collection of DaemonSet resources.                                          | DaemonSetStatus collection      |                                                                                                                      |
+| StatefulSetStatus collection testing: Test the status collection of StatefulSet resources.                                      | StatefulSetStatus collection    |                                                                                                                      |
+| PodDisruptionBudget collection testing: Test the status collection of PodDisruptionBudget resources.                            | PodDisruptionBudget collection  | [Resource Selector](https://karmada.io/docs/next/userguide/globalview/global-search-for-resources/#resourceselector) |
```

Comment:
I don't understand why the link about Resource Selector was posted, it doesn't seem to have a direct connection.

Reply:
in my view, most of the tests here are based on ResourceSelector.

Reply:
I understand what you mean, but we are not using the ResourceSelector of resource `ResourceRegistry`. I think we might need to remove this comment.

Reply:
ok, i understand